src/test/java/org/opentripplanner/smoketest/DenverSmokeTest.java

```diff
@@ -0,0 +1,35 @@
+package org.opentripplanner.smoketest;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.opentripplanner.smoketest.SmokeTest.assertThatItineraryHasModes;
+
+import java.util.List;
+import java.util.Set;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.transit.model.basic.WgsCoordinate;
+
+/**
+ * This smoke test expects an OTP installation running at localhost:8080
+ * <p>
+ * It uses the REST API to check that a route from South to North Denver can be found.
+ */
+@Tag("smoke-test")
+@Tag("denver")
+public class DenverSmokeTest {
+
+  WgsCoordinate southBroadway = new WgsCoordinate(39.7020, -104.9866);
+  WgsCoordinate twinLakes = new WgsCoordinate(39.8232, -105.0055);
+
+  @Test
+  public void routeFromSouthToNorth() {
+    var request = new SmokeTestRequest(southBroadway, twinLakes, Set.of("TRANSIT", "WALK"));
```

Comment:
var, interesting. What is the default JDK version for OTP now? This will break if merged into a branch where JDK 1.8 is used. It looks like ibi-dev-2.x is JDK 17 so its probably ok.

Reply:
OTP2/`ibi-dev-2.x` runs only on Java 17.

Reply:
Do we plan to merge these tests into OTP1/ibi-dev?

Reply:
I have no idea, probably not given the JDK version. @binh-dam-ibigroup thoughs?